plugin/clang.vim

```diff
@@ -767,6 +771,9 @@ func! s:ClangCompleteInit(force)
   " add include directories if is enabled and not ow
   if g:clang_include_sysheaders && ! l:is_ow
     let l:incs = s:DiscoverIncludeDirs(g:clang_exec, b:clang_options)
+    if g:clang_has_win
+      let l:incs += s:DiscoverIncludeDirs(g:gcc_exec, b:clang_options)
+    endif
```

Comment:
here may have problem, should test if `g:clang_gcc_exec` is executable before execute it. e.g. I don't have mingw on windows, but have the clang.exe.